src/serialize/column.rs

```diff
@@ -761,6 +783,7 @@ mod tests {
     enum ComponentId {
         Position,
         Velocity,
+        I32
```

Comment:
What purpose does adding an additional component type to these tests serve?

Reply:
Removed.